app/javascript/components/patient/utils/IconMinor.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
Can we call this file icons and move the `BadgeHoH` component there as well? Then you can just move the `HouseholdMemberTable` component into the main `patient/household` directory.

be sure to move the test files as well